### DISCUSSION TITLE
Nuxt 3 modules for tests?

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17948] [Questions] [answered]
What are the current tests for Nuxt 3?

**Comment by danielroe** [comment, marked as answer]
You might want to follow https://github.com/nuxt/framework/pull/2952

**Reply by mihail727** [reply]
I was trying to figure out how to use it.... but no result
Will the documentation expand in the future?

**Reply by TheAlexLichter** [reply]
As you can see, the PR is still in draft. Thus, the test module for Nuxt3 is still WIP. Follow the PR for more updates 😋 Docs will be added too when the module is ready.

**Reply by mihail727** [reply]
> As you can see, the PR is still in draft. Thus, the test module for Nuxt3 is still WIP. Follow the PR for more updates 😋 Docs will be added too when the module is ready.

Ok, thx
